tests/components/samsungtv/test_media_player.py

```diff
@@ -77,28 +77,30 @@ def tearDown(self):
 
     @MockDependency("samsungctl")
     @MockDependency("wakeonlan")
-    def test_setup(self, samsung_mock, wol_mock):
-        """Testing setup of platform."""
+    async def async_test_setup(self, samsung_mock, wol_mock):
```

Comment:
Tests need to be named `test_*` to be picked up as test functions by the test runner.

Reply:
We should write standalone pytest test functions instead of test methods of a test class.

See eg:
https://github.com/home-assistant/home-assistant/blob/8bd847ed3969e75a1c03dcf652df28ab65106d1e/tests/components/manual/test_alarm_control_panel.py#L83-L119